src/index.html

```diff
@@ -1,13 +1,72 @@
 <!DOCTYPE html>
-<html lang="en-US">
+<html lang="en" dir="ltr">
   <head>
-    <title>Webpack Starter Kit</title>
+    <meta charset="utf-8">
+    <title>Travel Tracker</title>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:400,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Sulphur+Point:400,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Webpack Starter Kit</h1>
-
-    <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
-
-    <!-- Do not include the index.js or main.bundle.js file here - it is done automatically by the webpack server -->
+    <section class="user-view" role="main">
+      <header class="main-header">
+          <h1 class="travel-tracker-logo">Travel Tracker</h1>
+          <h1 class="welcome-header">Welcome</h1>
+          <button type="button" class="sign-out-button" name="sign-out-button">Sign Out</button>
+      </header>
+      <header class="header-2">
+        <section class="year-travel-cost-widget" aria->
+          <h3 class="year-travel-cost-title">Year to Date Travel Cost</h3>
+          <p class="year-travel-Cost-Amount">$$$<p>
+        </section>
+        <h1 class="page-view-header">Trip History</h1>
+        <button class="plan-new-trip-button" type="button" name="button">Plan New Trip</button>
+        <button class="view-trip-history button hidden" type="button" name="button"></button>
+      </header>
+      <section class="trip-history-view">
+        <nav class="trip-by-time-button-navigator">
+          <button class="view-past-trips-button" type="button" name="past-trips-button">Past Trips</button>
```

Comment:
consider adding `view-trips-button` as a second class. This will allow you to easily make changes to these specific buttons simultaneously without altering any others.